app/src/main/java/org/schabi/newpipe/util/TextLinkifier.java

```diff
@@ -30,92 +37,175 @@ private TextLinkifier() {
+                                                         final StreamingService streamingService,
+                                                         final String contentUrl) {
         final Markwon markwon = Markwon.builder(context).usePlugin(LinkifyPlugin.create()).build();
         markwon.setMarkdown(textView, markdownBlock);
-        return changeIntentsOfDescriptionLinks(context, textView.getText(), textView);
+        return changeIntentsOfDescriptionLinks(context, textView.getText(), textView,
+                streamingService, contentUrl);
     }
 
+    private static final Pattern TIMESTAMPS_PATTERN_IN_PLAIN_TEXT =
+            Pattern.compile("(?:([0-5]?[0-9]):)?([0-5]?[0-9]):([0-5][0-9])");
+
     /**
-     * Change links generated by libraries in the description of a content to a custom link action.
+     * Add click listeners which opens the popup player on timestamps in a plain text.
      * <p>
-     * Instead of using an {@link android.content.Intent#ACTION_VIEW} intent in the description of a
-     * content, this method will parse the {@link CharSequence} and replace all current web links
+     * This method finds all timestamps in the {@link SpannableStringBuilder} of the description
+     * using a regular expression, adds for each a {@link ClickableSpan} which opens the popup
+     * player at the time indicated in the timestamps.
+     *
+     * @param context               the context to use
+     * @param spannableDescription  the SpannableStringBuilder with the text of the
+     *                              content description
+     * @param contentUrl            the URL of the content
+     * @param streamingService      the {@link StreamingService} of the content
+     */
+    private static void addClickListenersOnTimestamps(final Context context,
+                                                      final SpannableStringBuilder
+                                                              spannableDescription,
+                                                      final String contentUrl,
+                                                      final StreamingService streamingService) {
+        final String descriptionText = spannableDescription.toString();
+        final Matcher timestampMatches = TIMESTAMPS_PATTERN_IN_PLAIN_TEXT.matcher(descriptionText);
+
+        while (timestampMatches.find()) {
+            final int timestampStart = timestampMatches.start(0);
+            final int timestampEnd = timestampMatches.end(0);
+            final String parsedTimestamp = descriptionText.substring(timestampStart, timestampEnd);
+            final String[] timestampParts = parsedTimestamp.split(":");
+            final int seconds;
+            if (timestampParts.length == 3) { // timestamp format: XX:XX:XX
+                seconds = Integer.parseInt(timestampParts[0]) * 3600 + Integer.parseInt(
+                        timestampParts[1]) * 60 + Integer.parseInt(timestampParts[2]);
```

Comment:
For better readbility:
```suggestion
                seconds = Integer.parseInt(timestampParts[0]) * 3600 // hours
                        + Integer.parseInt(timestampParts[1]) * 60   // minutes
                        + Integer.parseInt(timestampParts[2]);       // seconds
```